### ISSUE TITLE
chore: get to 100% branch coverage

### ISSUE THREAD

**Issue by bcoe** [yargs/yargs#1465]
## Background

One of the other projects I've been working on is [c8](https://github.com/bcoe/c8), which collects coverage using V8's built in coverage (rather than using transpilation).

As of Node 13, enough bugs have been worked out in V8/c8 that we can actually hit 100% line coverage with yargs (and I've started [moving us over to dog-fooding c8](https://github.com/yargs/yargs/pull/1464)).

## Help Wanted

I would like for us to try to move towards 100% branch coverage with yargs, in the process of doing so we should file bugs against V8 for any issues we continue to find with their coverage support.

## Upstream Issues in V8

_we can file any outstanding issues against V8 here._

## Contributing

1. run the yargs test suite.
2. submit a patch adding a test for one of the untracked branches of coverage.

**Comment by nicolasacquaviva** [comment]
When I try to run `npm run coverage` I get `Bad response: 422 {"message":"Couldn't find a repository matching this job.","error":true}`. Am I missing something?

**Comment by bcoe** [comment]
@nicolasacquaviva `npm run coverage` is used on Travis to send coverage to coveralls.io. If you simply run `npm test`, it will collect and display coverage 👍

**Comment by dhuang612** [comment]
Is there anything left to work on?

Thanks

**Comment by bcoe** [comment]
@dhuang612 yeah :smile: there's still missing block coverage for quite a few statements, if you clone the repo and run tests, they should jump out at you.
